components/calendar/stories/calendar.stories.js

```diff
@@ -35,19 +38,33 @@ export default {
 			table: {
 				type: { summary: "datetime" },
 				category: "Content",
+				disable: true,
 			},
 			control: "date",
 			if: { arg: "isDisabled", truthy: false },
 		},
+		isRangeSelection : {
```

Comment:
I could understand if we wanted to remove this for now. It definitely feels like we could get this to work in the template, but I didn't want to sink too much time dedicated to that, in this PR.